net-analyzer/prelude-correlator/files/prelude-correlator.initd

```diff
@@ -0,0 +1,25 @@
+#!/sbin/runscript
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+BIN_CORRELATOR=/usr/bin/prelude-correlator
+PID_CORRELATOR=/run/prelude-correlator/prelude-correlator.pid
+
+depend() {
+  need net
```

Comment:
no because you can install prelude-correlator on one computer and prelude-manager on another